api_backend_server/tests/integration_tests/test_activities_apis.py

```diff
@@ -0,0 +1,30 @@
+import pytest
+import requests
+from collections import namedtuple
+
+@pytest.fixture(scope="module")
+def credentials():
+    Credentials = namedtuple("Credentials", "name category_id activity_start activity_end")
+    credentials = Credentials(
+        name="dummy_activity",
+        category_id=1,
+        activity_start="2020-04-30 09:34:35.414677",
```

Comment:
It's not a complete ISO-8601 format, as it's missing timezone information - example of an ISO-8601 formatted date: *2020-05-01T11:32:18,528057689+02:00*
ISO-8601 dates can be easily generated e.g. with bash *date* command: `date --iso-8601=ns`